littlelines/otwarchive
# Ao3 5883 settings

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5883

## Purpose

What does this PR do?
Puts restrictions in place for which admins can adjust which settings in the admin settings page.

## Testing Instructions

Sign in with an admin user with no rules and ensure the settings button is missing from the header in the admin section.

Sign in with a super admin and ensure that you can update all of the settings.

Sign in with an admin with only the tag_wrangling role, and ensure that you can only update the "Turn off tag wrangling for non-admins" setting.

Sign in with an admin with only the support role, and ensure that you can only update the "Turn off support form" setting.

Sign in with an admin with only the policy_and_abuse role, and ensure that you can only update the "Automatically hide spam works" setting.



(If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions.)

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

(If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

(If you do not fill in this section, we will use your GitHub account name and
they/them pronouns to avoid making assumptions about your gender.)
